language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/TomlCodeActionRouter.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ballerinalang.langserver.codeaction;
+
+import org.ballerinalang.langserver.codeaction.extensions.K8sDiagnosticsBasedCodeAction;
+import org.ballerinalang.langserver.commons.CodeActionContext;
+import org.eclipse.lsp4j.CodeAction;
+import org.eclipse.lsp4j.Diagnostic;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.ServiceLoader;
+
+/**
+ * Code Action Router for Kubernetes.toml.
+ *
+ * @since 2.0.0
+ */
+public class TomlCodeActionRouter {
```

Comment:
Let's rename this into `K8sTomlCodeActionRouter` since we are only dealing with `kubernetes.toml`